taxi/forms.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.forms import ModelForm, CheckboxSelectMultiple
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number"
+        )
+
+
+class DriverLicenseUpdateForm(ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("first_name", "last_name", "license_number")

```

Comment:
The `DriverLicenseUpdateForm` should also validate the `license_number` field to ensure it meets the format of 3 uppercase letters followed by 5 digits. This validation is necessary for both creating and updating a driver.